Pod/Classes/MenuItemView.swift

```diff
@@ -18,6 +18,16 @@ public class MenuItemView: UIView {
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
+    
+    lazy public var descLabel: UILabel = {
+        let label = UILabel(frame: .zero)
+        label.numberOfLines = 1
+        label.textAlignment = .Center
+        label.userInteractionEnabled = true
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
```

Comment:
Can you make a common method for initializing UILabel?
It's redundant since its exactly same code with `titleLabel.`